### PR TITLE
Set default payment account for an edited offer to null

### LINKED ISSUE #1565: Bug with fixed currency at edit offer
After  a user has clicked edit offer the currency will stick in create offer views as well as in the edit offer view.
To reproduce create 2 offers with fiat and an altcoin. Then go to edit offer at both and back to create offer. Switch payment method at create offer. The currency does not change anymore. At the edit offer it shows the same currency for both offers. 

See also: https://bisq.community/t/create-buy-offer-wrong-currency/5573/11

### PR DESCRIPTION
**Pull request by ripcurlx**
Fixes #1565.

### CODE REVIEW

File: src/main/java/bisq/desktop/main/offer/EditableOfferDataModel.java [hunk 1]
```diff
@@ -245,7 +245,7 @@ public boolean initWithData(OfferPayload.Direction direction, TradeCurrency trad
         fillPaymentAccounts();
 
         PaymentAccount account;
-        PaymentAccount lastSelectedPaymentAccount = preferences.getSelectedPaymentAccountForCreateOffer();
+        PaymentAccount lastSelectedPaymentAccount = getPreselectedPaymentAccount();
```
Comment (ManfredKarrer) [inline src/main/java/bisq/desktop/main/offer/EditableOfferDataModel.java]: Can we add @Nullable annotations for all nullable fields?

**Review by ManfredKarrer — APPROVED** [review]
utACK

I don't have enough context to reason about the change. Will checkout the code once its complete and review all again.

File: src/main/java/bisq/desktop/main/offer/EditableOfferDataModel.java (hunk 1 above)
Comment (ripcurlx) [reply src/main/java/bisq/desktop/main/offer/EditableOfferDataModel.java]: Done

**Review by ManfredKarrer — APPROVED** [review]
utACK

**Pull request merged**